script_editor/app/views/plays/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
-<html><body>
+<html manifest="MND.appcache"><body>
```

Comment:
Why are you creating a manifest for the html? Was this something leftover from saving? Where / what is "MND.appcache"